### ISSUE TITLE
Does texture(tex, uv) propagate any gradients to uv ?

### ISSUE THREAD

**Issue by yataoz** [NVlabs/nvdiffrast#20]
Hi, 

I have a neural net that outputs a uv map, which will be used as input to the texture sampling op. I wonder if the texture op passes any gradients to the uv input during training? Thanks.

**Comment by s-laine** [comment]
Sorry for the late reply &mdash; yes, the texture op propagates gradients to the input `uv` tensor as you would expect. If using sampling mode `'nearest'`, these gradients will be zero because the sampling result is piecewise constant, but all other sampling modes involve interpolation between texels and therefore produce gradients for the texture coordinates.
